src/pages/EditReportFieldPage.tsx

```diff
@@ -73,44 +84,78 @@ function EditReportFieldPage({route, policy, report}: EditReportFieldPageProps)
         Navigation.dismissModal(report?.reportID);
     };
 
+    const handleReportFieldDelete = () => {
+        ReportActions.deleteReportField(report.reportID, reportField);
+        Navigation.dismissModal(report?.reportID);
+    };
+
     const fieldValue = isReportFieldTitle ? report.reportName ?? '' : reportField.value ?? reportField.defaultValue;
 
-    if (reportField.type === 'text' || isReportFieldTitle) {
-        return (
-            <EditReportFieldTextPage
-                fieldName={Str.UCFirst(reportField.name)}
-                fieldKey={fieldKey}
-                fieldValue={fieldValue}
-                isRequired={!reportField.deletable}
-                onSubmit={handleReportFieldChange}
-            />
-        );
+    const menuItems: ThreeDotsMenuItem[] = [];
+
+    const isReportFieldDeletable = reportField.deletable && !isReportFieldTitle;
```

Comment:
`isReportFieldTitle` checks for formula also and the tile field can be text as well. So, we need to check if fieldID is title. More details here #49077